lib/IRGen/GenDistributed.cpp

```diff
@@ -391,12 +392,12 @@ FunctionPointer DistributedAccessor::getPointerToMethod() const {
   auto signature = IGM.getSignature(fnType, fpKind.useSpecialConvention());
 
   auto *fnPtr =
-      IGM.getAddrOfSILFunction(Method, NotForDefinition,
-                               /*isDynamicallyReplaceable=*/false,
-                               /*shouldCallPreviousImplementation=*/false);
+    llvm::ConstantExpr::getBitCast(IGM.getAddrOfAsyncFunctionPointer(Method),
+                                   signature.getType()->getPointerTo());
```

Comment:
ah so that's the "we need an async function pointer as function pointer" bit...

Reply:
Yeah…